### PR TITLE
Added combat outdoorsmanship

### PR DESCRIPTION
**Pull request by adamgoerdt**

### CODE REVIEW

File: combat-trainer.lic [hunk 1]
```diff
@@ -1211,6 +1223,11 @@ class TrainerProcess
       bput('meditate research turtle', 'You clear ')
     when 'Barb Research Debilitation'
       bput('meditate research earthquake', 'You clear ')
+    when 'Collect'
+      retreat
+      collect('rock')
```
Comment (rpherbig) [inline combat-trainer.lic]: Does `collect` correctly handle the case in which your room won't let you collect rocks?

File: combat-trainer.lic [hunk 2]
```diff
@@ -1211,6 +1223,11 @@ class TrainerProcess
...
+      waitrt?
+      fput('kick pile')
```
Comment (rpherbig) [inline combat-trainer.lic]: I'd prefer this to use `bput` if possible.

**Comment by rpherbig** [comment]
I'm seeing some unrelated changes in this PR. Can you look into that?

**Comment by adamgoerdt** [comment]
I don't know why github isn't merging these files correctly.  The unrelated lines are in your master repository as well as the file I'm trying to submit, but based on the compare github seems to think they're not in the master repo.  Any ideas on how to fix this?  I'm stumped

**Comment by rpherbig** [comment]
It sounds like you need to merge master back into your fork.

**Comment by adamgoerdt** [comment]
That did it.  Thanks

File: combat-trainer.lic (hunk 1 above)
Comment (adamgoerdt) [inline combat-trainer.lic]: It does now

**Pull request merged**